apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryClientImplTest.cs

```diff
@@ -556,5 +556,21 @@ public void ListTablesConversion()
             Assert.Equal("VIEW", tableResource.Type);
             Assert.True(tableResource.View.UseLegacySql);
         }
+
+        [Fact]
+        public void InsertRows_NoRows_NoOp()
+        {
+            var service = new FakeBigqueryService();
+            var client = new BigQueryClientImpl("project", service);
+            client.InsertRows("dataset", "table", new BigQueryInsertRow[0]);
```

Comment:
There are no Asserts. Will it throw an exception on test failure? Might be worth a comment.

Reply:
Yup, will add a comment.